queue_services/entity-pay/src/entity_pay/resources/worker.py

```diff
@@ -106,13 +106,13 @@ def worker():
 
     # None of these should bail as the filing has been marked PAID
     cloud_event = SimpleCloudEvent(
-        source=__name__[: __name__.find(".")],
+        source="business_pay",
```

Comment:
as the source is now not entity_pay but the project's name has actually changed, i had to put this manually.